yb-voyager/src/query/queryissue/parser_issue_detector_test.go

```diff
@@ -1595,7 +1595,7 @@ func TestTimestampOrDateHotspotsIssues(t *testing.T) {
 	parserIssueDetector.FinalizeColumnMetadata()
 
 	for stmt, expectedIssues := range sqlsWithExpectedIssues {
-		issues, err := parserIssueDetector.GetAllIssues(stmt, ybversion.LatestStable)
+		issues, err := parserIssueDetector.GetAllIssues(stmt, ybversion.V2024_2_3_1)
```

Comment:
Just confirming: If we keep it latest stable, then we can't test the detection logic because they won't be returned, right?